### PR TITLE
Trader joe subgraph

### PR DESCRIPTION
**Pull request by glupta**
Deployed branch to thegraph already.

Added VSP for Vesper
Added Joe and the following Trader Joe LB V2 LPs:
AVAX-USDC
BTCb-AVAX
WBTCe-BTCb
AVAX-USDT
USDT-USDC
USDCe-USDC
USDTe-USDT
BTCb-USDC
DAIe-USDC
sAVAX-AVAX
JOE-AVAX

Added `generated` and `build` directories to `.gitignore` to keep repo light

### CODE REVIEW

File: savvy-fairlaunch-avax-frens/build/AVAXAave/abis/ERC20.json
Comment (rkhadder) [inline savvy-fairlaunch-avax-frens/build/AVAXAave/abis/ERC20.json:1]: Why did you delete the build folder?

File: savvy-fairlaunch-avax-frens/schema.graphql [hunk 1]
```diff
@@ -25,9 +25,24 @@ type User @entity {
...
+  everHeldTJBTCbAVAX: Boolean!
+  everHeldTJJOEAVAX: Boolean!
+  everHeldTJWBTCeBTCb: Boolean!
+  everHeldTJAVAXUSDT: Boolean!
+  everHeldTJUSDTUSDC: Boolean!
+  everHeldTJUSDCeUSDC: Boolean!
+  everHeldTJUSDTeUSDT: Boolean!
+  everHeldTJBTCbUSDC: Boolean!
+  everHeldTJDAIeUSDC: Boolean!
+  everHeldTJsAVAXAVAX: Boolean!
```
Comment (rkhadder) [inline savvy-fairlaunch-avax-frens/schema.graphql:47]: You need to initialize these when retrieving the user other sync will fail.

**Review by rkhadder — CHANGES_REQUESTED** [review]

File: savvy-fairlaunch-avax-frens/build/AVAXAave/abis/ERC20.json
Comment (glupta) [reply savvy-fairlaunch-avax-frens/build/AVAXAave/abis/ERC20.json:1]: because we recreate this locally

**Review by rkhadder — APPROVED** [review]
